### ISSUE TITLE
[Bug] SAGEConv Fails to Handle Graph without Edges

### ISSUE THREAD

**Issue by toooooodo** [dmlc/dgl#1754]
## To Reproduce

```python
g = dgl.DGLGraph()
g.add_nodes(5)
model = SAGEConv(in_feats=5, out_feats=5, aggregator_type='mean')
output = model(g, torch.randn([5, 5]))
```

get error `KeyError: 'neigh'`
## Expected behavior
In the above case, there is no edge in graph, so the message passing process does not take place and `neigh` is not in nodes' features. But it seems unreasonable to report an error.

## Environment

 - DGL Version (e.g., 1.0): 0.4.3
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): PyTorch 1.4.0
 - Python version: 3.7

**Comment by mufeili** [comment]
Hi, thank you for the report. I agree this should be fixed.

**Comment by mufeili** [comment]
This shall be addressed in #1792 and you can have that feature by installing from source.
